packages/main/src/Avatar.ts

```diff
@@ -331,7 +331,7 @@ class Avatar extends UI5Element implements ITabbable {
 
 	get validInitials() {
 		// initials should consist of only 1,2 or 3 latin letters
-		const validInitials = /^[a-zA-Z]{1,3}$/,
+		const validInitials = /^[a-zA-Zà-üÀ-Ü]{1,3}$/,
```

Comment:
Perphaps to ensure matching every other letters than A–Z, we can use predefined character classes describing Unicode characters like :
- `\p{L}` or `\p{Letter}` for any kind of letter from any language. 
- `\p{M}` or `\p{Mark}`: a character intended to be combined with another character (e.g. accents, umlauts, enclosing boxes, etc.).

https://www.regular-expressions.info/unicode.html#prop

Example: `^[\p{L}\p{M}]{1,3}$`

Reply:
It is stated that the initials are supposed to accept only Latin letters - since the accented characters are part of Latin-1 char set, we should include them. 
With your example, we will match even languages, which are not build on an alphabet - for such cases the icon should be displayed.

Reply:
Valid.